Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PatientSearchBean.java

```diff
@@ -653,4 +663,31 @@ public void setDisplayOrganizationName(String displayOrganizationName) {
         this.displayOrganizationName = displayOrganizationName;
     }
 
+    /**
+     * @return the documentTypeName
+     */
+    public String getDocumentTypeName() {
+        //
+        if (getSelectedCurrentDocument().getDocumentType() != null) {
+            return getDocumentTypeNameFromTheStaticList(getSelectedCurrentDocument().getDocumentType());
+        }
+        return null;
+    }
+
+    private String getDocumentTypeNameFromTheStaticList(String documentType) {
+        for (SelectItem localDocumentTypeList : documentTypeList) {
+            if (localDocumentTypeList.getValue().equals(documentType)) {
+                return localDocumentTypeList.getLabel();
+            }
+        }
+        return "Unknown Document";
+    }
+
+    /**
+     * @return the documentInfoModalWindowHeader
+     */
+    public String getDocumentInfoModalWindowHeader() {
+        String localAuthorPerson = getSelectedCurrentDocument().getAuthorPerson() == null ? "Unknown" : getSelectedCurrentDocument().getAuthorPerson();
+        return getDocumentTypeName() + " for " + getSelectedCurrentPatient().getName() + " Created by " + localAuthorPerson + " on " + getCreationTimeUiDisplay();
```

Comment:
Can we shorten this to just doc title and patient name.  Looked a little strange when I displayed it.

Thinks it should just show: "Document Title, John Doe".

Kerry had expressed the same comment on this to me too.